.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,29 @@
+---
+name: Bug report
+about: Create a report to help us improve Zephyr
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
```

Comment:
I'd recommend adding a section describing the impact of this bug to the reporter, something like:

    **Impact**
    What impact does this issue have on your progress (e.g., annoyance, showstopper)